Add Getting Started Guide to UG

B-enguin opened the pull request:
Edit UG to include new guide for new users to get started with HomeBoss. Includes installation, application layout, command format, and short tutorial.

Edit CSS Styling to make code blocks more visible, begin migration to syntax highlighting for code blocks.

Update Hyperlink in Site Map for Getting Started.

Closes #283.

B-enguin committed:
Add Getting Started Guide to UG.

Edit UG to include new guide for new users to get started with HomeBoss. Includes installation,
application layout, command format, and short tutorial.

Edit CSS Styling to make code blocks more visible, begin migration to syntax highlighting for code
blocks.

Update Hyperlink in Site Map for Getting Started.

B-enguin committed:
Add Getting Started Guide to UG

Edit UG to include new guide for new users to get started with HomeBoss. Includes installation,
application layout, command format, and short tutorial.

Edit CSS Styling to make code blocks more visible, begin migration to syntax highlighting for code
blocks.

Update Hyperlink in Site Map for Getting Started.

B-enguin committed:
Fix EOF Newline

B-enguin committed:
Merge remote-tracking branch 'origin/283-ug-getting-started' into 283-ug-getting-started

# Conflicts:
#	docs/stylesheets/main.css

juliusgambe commented on docs/images/getting-started/HomePage.png:
I think it should be Customer/Delivery List *Panel*?

juliusgambe reviewed (commented):
I only looked through the annotations.

juliusgambe commented on docs/UserGuide.md:
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
+
+<box type="tip" background-color="#d9edf7" border-color="#bce8f1" icon=":bulb:">
+
+**Tip**: You may find the following links helpful should you need to install `Java 11`{.swift}
```
**Tip**: -> **Tip:**?

B-enguin committed:
Rename of Customer Delivery List Box to Customer Delivery List Panel

B-enguin committed:
Fix Styling for email syntax parsing

Change styling for hljs-meta to follow styling of regular text.

Update UserGuide.md Getting Started Images to have non-fixed sizing.

zhonghan721 commented on docs/UserGuide.md:
````diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
+```
+
+* The **Command Phrase** is `update`{.swift}
+* The **Parameters** are the words in `UPPER_CASE`{.swift}, for example `USERNAME`{.swift} or `PASSWORD`{.swift}
+* The **Prefixes** are special keywords that a preceded by `--`{.SWIFT}, such as `--user`{.swift} or
````
Is there a typo here (`that are preceded` and not `a`), and would `and` be more suitable as compared to `or`?

B-enguin replied:
Good catch

zhonghan721 commented on docs/UserGuide.md:
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
+* Items that are placed in square brackets are optional, e.g. `DELIVERY_ID [--name DELIVERY_NAME]`{.swift} can either be
+  `1`{.swift} or `1 --name deliveryName`{.swift}.
+* However, items that are grouped together in square brackets must all be provided together, 
+  e,g, for `[--password PASSWORD --confirmPass CONFIRM_PASSWORD]`{.swift} both `PASSWORD`{.swift} and 
```
Would it be `e.g.` instead?

B-enguin replied:
Good catch

zhonghan721 commented on docs/UserGuide.md:
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
+
+8. Type `delivery add furniture --customer CUSTOMER_ID --date 2023-12-03`{.swift}, replacing `CUSTOMER_ID`{.swift} with 
+   Gabriel's ID, and hit enter.
+9. new Delivery should have been added for Gabriel, you should see a new Delivery similar to the image below.
```
Should it start as a sentence?

B-enguin replied:
Good catch

zhonghan721 reviewed (commented):
Some general comments

B-enguin committed:
Updates to UG based on comments

B-enguin committed:
Update UG based on comments

B-enguin committed:
Update UG to include repeatable parameters

B-enguin committed:
Merge branch 'master' into 283-ug-getting-started

B-enguin committed:
Update HomePage.png

B-enguin committed:
Merge remote-tracking branch 'origin/283-ug-getting-started' into 283-ug-getting-started
